test-projects/demo/dashboards/welcome.visivo.yml

```diff
@@ -71,9 +71,14 @@ dashboards:
           - width: 2
             table:
               name: Fibonacci Waterfall Table
-              trace: ref(Fibonacci Waterfall Trace)
+              traces:
```

Comment:
Do we want to scope in allowing people to create tables directly from models as well?

Reply:
I do think that is a good idea, but maybe a follow up?